support/reattach-pv/main.go

```diff
@@ -28,6 +30,7 @@ import (
 
 const (
 	esManifestFlag = "elasticsearch-manifest"
+	oldEsNameFlag  = "old-elasticsearch-name"
```

Comment:
I'm still a bit confused by this flag:
* Its value is never used, we could replace it by `viper.GetBool(oldEsNameFlag)`
* The 2nd parameter in `expectedVolumeClaims` is not used

Reply:
Actually, I see your point now @barkbay , and this brings up a concern about matching exactly the PVs of the old cluster.  I'm going to revisit your comment about making the regex directly dependent on the old ES name.

Reply:
Commit done, and verified with a new round of testing @barkbay .  The `oldEsName` is now used in the regex to limit the volumes that are used.